### PR TITLE
docs: add faq section explaining why rclone changes fullwidth characters in file names

### PR DESCRIPTION
**Pull request by albertony**
#### What is the purpose of this change?

Title says it all.

#### Was the change discussed in an issue or in the forum before?

https://forum.rclone.org/t/cannot-correctly-handle-filenames-with-u-ff1a-full-width-colon/26698

#### Checklist

- [X] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [ ] ~I have added tests for all changes in this PR if appropriate.~
- [X] I have added documentation for the changes if appropriate.
- [X] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [X] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: docs/content/faq.md [hunk 1]
```diff
@@ -210,3 +210,17 @@ The most common cause of rclone using lots of memory is a single
 directory with thousands or millions of files in.  Rclone has to load
 this entirely into memory as rclone objects.  Each rclone object takes
 0.5k-1k of memory.
+
+### Rclone changes fullwidth characters in file names
+
+To handle varying sets of [restricted filenames](/overview/#restricted-filenames)
+on different backends, rclone transparently converts between restricted
+characters and similar looking Unicode characters. If you have file names
```
Comment (ncw) [inline docs/content/faq.md:218]: I think we should have an example here with `:` and its unicode equivalent.

**Review by ncw — COMMENTED** [review]
Very nice docs - thank you :-)

I think we need an example in the FAQ, but otherwise perfect!

**Comment by albertony** [comment]
New attempt! But I'm getting quite 😵 , trying to write explanation in one place (e.g. the faq), then improve some another place (e.g. in the overview), just to end up writing almost the same (and too long) section both places😫 ! But this encoding stuff can be quite confusing to users - and me, so I feel if I try to keep it short and concise then there are nuances that get lost that may well be exactly the thing that made it blow up for some user..

Well... Have a read, and let me know if I should rewrite, restructure or fix any nitpicks, and I will take another round 👨‍🌾

**Comment by albertony** [comment]
Given the new information, I "promoted" the detailed explanation originally written in faq into a new section `Caveats` in the overview page. I also moved and merged large sections from the `Encoding example: Windows`. I think this is better - less duplication. But perhaps the "negatively charged" `Caveats` section now gets *too* much attention? What do you think, @ncw?

**Comment by ncw** [comment]
...I think I'm going to merge this for 1.57 as it is a great improvement. We can do further followups to tweak stuff :-)

**Pull request merged**